code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -350,6 +350,38 @@ var/global/list/bottle_colour_choices = list("Blue" = "#0094FF","Dark Blue" = "#
 /obj/item/weapon/storage/pill_bottle/sweets/strange
 	items_to_spawn = list(/obj/item/weapon/reagent_containers/food/snacks/sweet/strange = 10)
 
+/obj/item/weapon/storage/pill_bottle/syndiemints
+	name = "box of mints"
+	desc = "Gets rid of halitosis and satisfied customers in one go! You shouldn't be seeing this."
+	icon = 'icons/obj/candymachine.dmi'
+	icon_state = "mintboxgeneric"
+	storage_slots = 50
+	items_to_spawn = list(/obj/item/weapon/reagent_containers/food/snacks/mint/syndiemint = 50)
+
+/obj/item/weapon/storage/pill_bottle/sweets/New()
```

Comment:
Did you hear about the coder who pasted a path from another storage item and forgot to change it?

He was destroyed!

Reply:
oh no
I don't wanna be destroyed! Fixed.